muikku-core-plugins-persistence/src/main/java/fi/otavanopisto/muikku/plugins/announcer/model/Announcement.java

```diff
@@ -143,6 +156,10 @@ public void setPinned(boolean pinned) {
   @Column (nullable=false)
   private Boolean publiclyVisible;
   
+  @ManyToMany (cascade = CascadeType.ALL)
```

Comment:
Onko cascade testattu? Ihan vaan ettei käy niin, että jos/kun kategoria poistetaan niin koko Announcement poistuu samalla.

Ks. esim https://stackoverflow.com/questions/4935095/jpa-hibernate-many-to-many-cascading